### ISSUE TITLE
s4c.cymru: detection

### ISSUE THREAD

**Issue by KarlG1965** [uBlockOrigin/uAssets#15667]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)
- [X] I verified the issue is not caused by the browser built-in blocker, or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through VPN)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.s4c.cymru/clic/programme/515091849`

### Category

detection

### Describe the issue

When I load the page it shows me a message saying 'adblocker detected'

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.45.0
Firefox: 106
filterset (summary): 
  network: 91316
  cosmetic: 42836
  scriptlet: 16697
  html: 797
listset (total-discarded, last-updated): 
  default: 
    user-filters: 478-12, never
    ublock-filters: 31955-58, 1d.10h.22m
    ublock-badware: 3972-0, 1d.10h.22m
    ublock-privacy: 286-0, 1d.10h.22m
    ublock-abuse: 76-0, 1d.10h.22m
    ublock-unbreak: 1909-1, 1d.10h.22m
    easylist: 64278-617, 1d.10h.22m
    easyprivacy: 30820-1720, 1d.10h.22m
    urlhaus-1: 10191-0, 10h.22m
    plowe-0: 3670-3, 1d.10h.22m
    DEU-0: 6020-2, 10h.21m
    ublock-quick-fixes: 566-0, 10h.20m
filterset (user): [array of 478 redacted]
trustedset: 
  added: [array of 6 redacted]
modifiedUserSettings: 
  advancedUserEnabled: true
  showIconBadge: false
  tooltipsDisabled: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 172 ms (selfie)
  maxAssetCacheWait: 218 ms
popupPanel: 
  blocked: 2
  blockedDetails: 
    s4c.cymru: 1
    googletagmanager.com: 1
```
</details>

**Comment by ghajini** [comment]
does this work
`@@||s4c.cymru/clic/js/ads-prebid.js$script,1p`

**Comment by KarlG1965** [comment]
Hi,
        
        Unfortunately not, no.
        
        I added this to my filters, refreshed and still got the
        following
        
        
        
        Karl
      
    Am 14.11.22 um 09:52 schrieb ghajini:
    
    
      
      does this work
        @@||s4c.cymru/clic/js/ads-prebid.js$script,1p
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you authored the thread.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/uBlockOrigin/uAssets/issues/15667#issuecomment-1313303790",
"url": "https://github.com/uBlockOrigin/uAssets/issues/15667#issuecomment-1313303790",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by mapx-** [comment]
No filter necessary on my end, all ok.

@KarlG1965 
- disable any other addon
- update uBO to the last build: https://github.com/gorhill/uBlock/releases/tag/1.45.3b1
- force update ALL filter lists
- test again

**Comment by ghajini** [comment]
Detection reproduced after login

**Comment by mapx-** [comment]
yes, I have an account, using uk ip and no detection on my end

**Comment by KarlG1965** [comment]
Sorry for the late reply.
        
        I confirm that after updating to the latest version and manually
        refreshing the filter lists uBlock
        is no longer detected.
        
        Many thanks for the fix guys!
        
        Karl
      
    Am 15.11.22 um 15:54 schrieb mapx:
    
    
      
      Closed #15667 as completed.
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you were mentioned.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/uBlockOrigin/uAssets/issues/15667#event-7814774006",
"url": "https://github.com/uBlockOrigin/uAssets/issues/15667#event-7814774006",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
